lib/errors/course_error_records.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Errors
```

Comment:
Why is this a module nested within a module?

Reply:
I started with a non-nested module, but I could find less examples and got influenced by the convention followed by the other modules in `lib/errors` so I did this. I'll remove the outer module.